aqua-ethos-setup/aqua_ethos.sh

```diff
@@ -123,6 +146,94 @@ function makePut {
 	fi
 }
 
+function getGatewayid {
+    GATEWAY=$(makeGET servers body| jq "[.[].id]")
+}
+
+function createLabel_if_does_not_exist {
+    CREATE_LABEL=$1
+    ENCODED_LABEL=${CREATE_LABEL// /%20}
+    JQ_LABEL_FILTER='.[] | select([.name=="'$CREATE_LABEL'"] | any) | .name'
+    RESPONSE_LABEL=$(makeGet settings/labels body  | jq "$JQ_LABEL_FILTER")
+    if [ -n "$RESPONSE_LABEL" ]; then
+        log "Label \"$CREATE_LABEL\" already exists."
+    else
+        log "Label \"$CREATE_LABEL\" does not exist, creating it."
+        NEW_LABEL=$(makePost settings/labels "{ \"name\": \"$CREATE_LABEL\" }" | jq ".name")
+        if [ "$NEW_LABEL"="$CREATE_LABEL" ]; then
```

Comment:
Not sure about this if statement - seems like it checks the response is equal the request, but does not do a lot if it does not match. Maybe at least have a log?